integration-tests/aws-lambda-test/src/main/java/co/elastic/test/TestLambda.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.test;
+
+import co.elastic.apm.api.ElasticApm;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+public class TestLambda implements RequestHandler<String, String> {
+    @Override
+    public String handleRequest(String command, Context context) {
+        ElasticApm.currentTransaction().addCustomContext("command", command);
+        if (command.startsWith("sleep ")) {
+            long millis = Long.parseLong(command.substring("sleep ".length()));
+            doSleep(millis);
```

Comment:
feels like this parse needs error handling for misconfigured comands

Reply:
What would an ideal solution look like for you?
E.g. a lambda with exactly the following behaviour regarding commands:
 * `greet XYZ` -> returns "Hello XYZ".
 * `sleep A` -> sleeps "A" amount of millis, returns empty string
 * `die` -> kills the Lambda JVM
 * everything else -> throws an error

Reply:
yes, why not tie it down tight, it's testing so we want to see anything odd